definitions/types/email.json

```diff
@@ -0,0 +1,10 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "type": "string",
+  "format": "email",
```

Comment:
learned about this today, `format` has some built-in checks, email regex is actually quite complicated and they handle it for us.

https://json-schema.org/understanding-json-schema/reference/string.html#built-in-formats

Reply:
Love it!

Reply:
🍾